Source/Python/pyWarpX.cpp

```diff
@@ -67,6 +69,8 @@ PYBIND11_MODULE(PYWARPX_MODULE_NAME, m) {
     init_MultiParticleContainer(m);
     init_WarpX(m);
 
+    m.def("run_div_cleaner", &run_ProjectionDivCleaner);
```

Comment:
Adding the Python binding here is a bit different from how similar things are done currently. The more standard approach would be that `run_ProjectionDivCleaner()` be a member function of `WarpX`, and the Python binding would then go in `Source/Python/WarpX.cpp`. Do you think it is worth making the change to follow that standard?

Reply:
Done